pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -481,6 +481,12 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 					Verbs:     sets.NewString("get"),
 					Resources: sets.NewString("secrets"),
 				},
+				{
+					// TODO: restrict to configmaps used by pods scheduled on bound node once supported
+					// Needed for configmap volume and envs
+					Verbs:     sets.NewString("get"),
+					Resources: sets.NewString("configmaps"),
```

Comment:
I would group this with the secrets rule above, since they need the same permissions for the same reasons